content/en/tracing/profiler/enabling/ddprof.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: Enabling the Datadog Profiler for Linux
+title: Enabling the Profiler for Compiled Languages
 kind: Documentation
-code_lang: linux
+code_lang: ddprof
 type: multi-code-lang
 code_lang_weight: 90
+aliases:
+  - /tracing/profiler/enabling/linux/
```

Comment:
Just to make sure:  this alias exists to make sure old links stay active?

Reply:
Correct